app/views/sign_up/registrations/new.html.erb

```diff
@@ -10,7 +10,7 @@
       label: t('account.login.tab_navigation'),
       routes: [
         { text: t('links.sign_in'), path: new_user_session_url },
-        { text: t('links.create_account'), path: sign_up_email_path },
+        { text: t('links.create_account'), path: sign_up_email_url },
```

Comment:
Minor: I might actually think it could be better to standardize in the other direction, so that the `path` we're logging is an actual path and not a fully-qualified URL?